jobbergate-cli/jobbergate_cli/subapps/pagination.py

```diff
@@ -0,0 +1,95 @@
+from typing import Any, Dict, List, Optional, cast
+
+import inquirer
+
+from jobbergate_cli.constants import PaginationChoices
+from jobbergate_cli.render import StyleMapper, render_paginated_list_results
+from jobbergate_cli.requests import make_request
+from jobbergate_cli.schemas import JobbergateContext, ListResponseEnvelope
+
+
+def handle_pagination(
+    jg_ctx: JobbergateContext,
+    url_path: str,
+    abort_message: str = "There was an error communicating with the API",
+    params: Optional[Dict[str, Any]] = None,
+    title: str = "Results List",
+    style_mapper: StyleMapper = None,
+    hidden_fields: Optional[List[str]] = None,
+):
+    assert jg_ctx is not None
+    assert jg_ctx.client is not None
+
+    current_page = 1
+
+    while True:
+        envelope = cast(
+            ListResponseEnvelope,
+            make_request(
+                jg_ctx.client,
+                url_path + f"?page={current_page}",
```

Comment:
I think that `page` could be added on `params` to make the code more readable, do you agree?